api/scpca_portal/test/management/commands/test_create_portal_metadata.py

```diff
@@ -51,19 +52,41 @@ def test_create_portal_metadata(self):
         self.load_test_data()
         self.processor.create_portal_metadata(clean_up_output_data=False)
 
-        with ZipFile(common.OUTPUT_PORTAL_METADATA_ZIP_FILE_PATH) as zip:
+        zip_file_path = ComputedFile.get_local_file_path(
+            common.GENERATED_PORTAL_METADATA_DOWNLOAD_CONFIG
+        )
+        with ZipFile(zip_file_path) as zip:
             # Test the content of the generated zip file
-            # There is 1 file:
+            # There are 2 file:
             # ├── README.md
-            expected_file_count = 1
+            # |── metadata.tsv
+            expected_file_count = 2
             # The filenames should match the following constants specified for the computed file
-            expected_file = readme_file.OUTPUT_NAME
+            expected_files = {
+                readme_file.OUTPUT_NAME,
+                metadata_file.MetadataFilenames.METADATA_ONLY_FILE_NAME,
+            }
             files = set(zip.namelist())
             self.assertEqual(len(files), expected_file_count)
-            self.assertIn(expected_file, files)
+            self.assertEqual(files, expected_files)
```

Comment:
We can remove both of these checks since we are already asserting that they are in the zip.namelist immediately after this. You can assert that it is 2 files if you want to ensure there are no extraneous files in the output zip.

Reply:
See at 651d753c49c931eb71237155e64ad99226d6e9c8.